hummingbot/connector/derivative/perpetual_finance/perpetual_finance_derivative.py

```diff
@@ -550,6 +553,13 @@ async def _funding_info_polling_loop(self):
                 funding_infos = await safe_gather(*funding_info_tasks, return_exceptions=True)
                 for trading_pair, funding_info in zip(self._trading_pairs, funding_infos):
                     self._funding_info[trading_pair] = funding_info["fr"]
+                    self._funding_info[trading_pair] = FundingInfo(
```

Comment:
This line assigns a value to `self._funding_info[trading_pair]` after in the previous line a different value was assigned to the same place. It seems there is an error in either of the two statements.